app/views/events/participants.html.erb

```diff
@@ -36,6 +36,6 @@
   </table>
   
   
-  <%= link_to t('.back', :default => t("helpers.links.back")),
+  <%= link_to t('.back', :default => t('helpers.links.back')),
```

Comment:
This is what I heard as well, double quoted strings should only be used for string interpolation